commercial/app/views/hosted/guardianHostedArticle.scala.html

```diff
@@ -1,176 +1,238 @@
+                                <div class="animal__stripe-even">
+                                @fragments.image(
+                                    ImageMedia.make(Seq(ImageAsset.make(Asset(AssetType.Image, Some("image/jpeg"), Some(page.customData.gazellePic)), 0))),
+                                    Seq("animal__stripe-character"),
+                                    GalleryMedia.inline,
+                                    "Zootropolis The pop star"
+                                )
+                                </div>
                                 <div class="animal__info">
                                     <div class="animal__info-poster">
                                         <img src="@{page.customData.gazellePoster}" alt="@{page.campaign.owner}">
                                     </div>
                                     <div class="animal__info-about">
-                                        <h3 class="animal__info-about-header hosted-tone--zootropolis">The Pop Star</h3>
-                                        <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut eget velit efficitur, viverra velit vel, vehicula sapien. Duis eu ligula augue. Donec sed ligula at massa bibendum sodales. Aenean quis lorem auctor, semper sem at, interdum ligula. Mauris lacinia pharetra augue ac gravida.</span>
-                                        <a href="#" class="animal__info-about-download">Download character poster</a>
+                                        <h3 class="animal__info-about-header hosted-tone--zootropolis">Gazelle - The pop star</h3>
+                                        <p class="text">
+                                            <i>“Try everything.“</i>
+                                        </p>
+                                        <p class="text">
+                                            Gazelle is very popular in the world of Zootropolis, having many fans, including Judy Hopps and Clawhauser. Her importance and popularity stem from the fact that she is very socially minded and is all about acceptance and equality for all.</span>
+                                        </p>
+                                        <p>
+                                            <a href="@{page.customData.posterPdf}" class="animal__info-about-download" target="_blank">Download character poster</a>
+                                        </p>
                                     </div>
                                 </div>
-                            </div>
+                            </div><!--End of animals !-->
+
+                            <div class="hosted__mobile">
+                            @for(nextPage <- page.nextPage) {
+                                <div class="hosted__next-video">
+                                    <div class="hosted__next-video--header">
+                                        <h2 class="hosted__text hosted__next-video--up-next">More from</h2>
```

Comment:
From an accessibility POV, it would be much better to use a single `H2` and then `SPAN` the hell out of these things:

```
<h2 class="hosted__text">
  <span class="hosted__next-video__up-next"></span>
  <span class="hosted__next-video__client-name"></span>
</h2>
```